### ISSUE TITLE
Multiple variables defined in a single line with initializers are not recognized.

### ISSUE THREAD

**Issue by felixhao28** [atom/language-java#178]
### Description

```java
class A {
    void f(int a) {
        int b = a + 1, c = a + 2;
```
![image](https://user-images.githubusercontent.com/2417391/49273674-bbfad700-f4b0-11e8-9f88-dedbcfa27ab3.png)

**Expected behavior:** [What you expect to happen]

`c` should be recognized as `variable.other.definition.java, meta.definition.variable.java` just like `b`.

**Comment by sadikovi** [comment]
Good catch. I don't think it was ever supported. Before I moved code from the variable definition, such variables would be assigned the parent scope and highlighted as such creating an impression that they were parsed as variables. And my change basically exposed the issue.

**Comment by felixhao28** [comment]
This also applies to field definition in class:

```java
public class AssetsKit {
    private static final Log log = Log.getLog(AssetsKit.class);
    private static final Charset UTF_8 = Charset.forName("UTF-8");
    private static final String JS_EXT = ".js", CSS_EXT = ".css";
    private static final String PROTOCOL = "^https?://.+$";
}
```

![image](https://user-images.githubusercontent.com/2417391/49414132-3bd4ba00-f7ad-11e8-955c-ee5f0aaccfe3.png)

**Comment by sadikovi** [comment]
I will investigate, thanks.
